homeassistant/components/jellyfin/media_player.py

```diff
@@ -149,7 +149,11 @@ def _update_from_session_data(self) -> None:
             media_content_type = CONTENT_TYPE_MAP.get(self.now_playing["Type"], None)
             media_content_id = self.now_playing["Id"]
             media_title = self.now_playing["Name"]
-            media_duration = int(self.now_playing["RunTimeTicks"] / 10000000)
+            media_duration = (
+                int(self.now_playing["RunTimeTicks"] / 10000000)
+                if "RunTimeTicks" in self.now_playing
+                else None
```

Comment:
Please avoid the ternary operator being across multiple lines. Instead, use a normal if-structure in that case.

Reply:
Done. I basically copied the other ternary statements in the original code. I'll submit a follow-up PR to change those statements too.